documentation/book/appendix_metrics.adoc

```diff
@@ -52,19 +52,33 @@ The following resources are defined:
 
 ==== Deploying on {OpenShiftName}
 
-To deploy all these resources you can run the following.  Note that this file creates a `ClusterRoleBinding` in the `myproject` namespace.  If you're not using this namespace then download the resource file locally and update it.
+The provided YAML file, with all the Prometheus related resources, creates a `ClusterRoleBinding` in the `myproject` namespace.
+If you're not using this namespace then download the resource file locally and update it as follow.
+
+[source,shell,subs=attributes+]
+curl -o prometheus.yaml https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/kubernetes.yaml
+sed -i 's/namespace: .\*/namespace: _my-namespace_/' prometheus.yaml
+
+To deploy all these resources you can run the following.
 
 [source,shell,subs=attributes+]
 oc login -u system:admin
-oc apply -f https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/kubernetes.yaml
+oc apply -f prometheus.yaml
 
 ifdef::Kubernetes[]
 ==== Deploying on {KubernetesName}
 
-To deploy all these resources you can run the following.  Note that this file creates a `ClusterRoleBinding` in the `myproject` namespace.  If you're not using this namespace then download the resource file locally and update it.
+The provided YAML file, with all the Prometheus related resources, creates a `ClusterRoleBinding` in the `myproject` namespace.
+If you're not using this namespace then download the resource file locally and update it as follow.
```

Comment:
If you are using a different namespace, download the resource file and update it as follows: